py/README.rst

```diff
@@ -13,7 +13,7 @@ Usage
 To install it execute::
 
     $ python3.5 -m venv venv
-    $ venv/bin/python -m pip install -r requirements.txt
+    $ venv/bin/python -m pip install -r py/requirements.txt
```

Comment:
But since the command is venv/bin/python .... means we are in the  ......../crate-stresstest "root" dir, or no?